%%
%% Copyright (c) 2018-2019 Weitian LI <wt@liwt.net>
%% CC BY 4.0 License
%%
%% Résumé
%% ------
%% A short document (1-2 pages) to sum up the job-related accomplishments
%% and experience.
%%
%% Checklist
%% ---------
%% * Contact Information
%% * Work History / Experience
%% * Education
%% * Skills
%% * Summary & Objective (optional)
%% * Hobbies & Interests (optional)
%%
%% Credits
%% -------
%% * CV vs. Resume: What is the Difference? When to Use Which?
%%   https://uptowork.com/blog/cv-vs-resume-difference
%% * How to Make a Resume: A Step-by-Step Guide (+30 Examples)
%%   https://uptowork.com/blog/how-to-make-a-resume
%% * Entry-Level Resume: Sample and Complete Guide (+20 Examples)
%%   https://uptowork.com/blog/entry-level-resume-example
%%
%% Created: 2018-04-14
%%

% English version
\documentclass{resume}

% Adjust icon size (default: same size as the text)
\iconsize{\Large}

% % File information shown at the footer of the last page
% \fileinfo{%
%   \faCopyright{} 2018--2020, Weitian LI \hspace{0.5em}
%   \creativecommons{by}{4.0} \hspace{0.5em}
%   \githublink{liweitianux}{resume} \hspace{0.5em}
%   \faEdit{} \today
% }

\name{Peng}{NI}

\keywords{Bioinformatics, Epigenetics, Python, R}

% \tagline{\icon{\faBinoculars}} <position-to-look-for>}
% \tagline{<current-position>}
\tagline{Lecturer}

\profile{
  \mobile{+86-152-7319-1121}
  \email{nipeng@csu.edu.cn}
  \github{PengNi} \\
  \university{School of Computer Science and Engineering, Central South University}
  \degree{PhD in Computer Science and Technology} \\
  \birthday{1991-12-26}
  \address{Changsha \textbullet Hunan}
  % Custom information:
  % \icontext{<icon>}{<text>}
  % \iconlink{<icon>}{<link>}{<text>}
}

\photo{9em}{nipeng_csu.jpg}

\begin{document}
\makeheader

%======================================================================
% Summary & Objectives
%======================================================================
{\onehalfspacing\hspace{2em}%
I got a PhD degree majored in bioinformatics, under the supervision of Prof. Jianxin Wang. Currently I am focusing on developing computitional methods for base modification detection using long read sequencing. During my PhD studies, I developed tools like \link{https://github.com/bioinfomaticsCSU/deepsignal}{DeepSignal}, \link{https://github.com/PengNi/deepsignal-plant}{DeepSignal-plant}, and \link{https://github.com/PengNi/ccsmeth}{ccsmeth} for 5mC detection.
\par}

%======================================================================
\sectionTitle{Competences \& Languages}{\faWrench}
%======================================================================
\begin{competences}[10em]
  \comptence{Programming}{%
    Python, R/ggplot2, Java, Nextflow, Rust (learning)
  }
  \comptence{Tools}{%
    SSH, Git, Bash/Shell
  }
  \comptence{Languages}{
    \textbf{English} ---
      reading \& writing (good), 
      listening \& speaking (conversant)
  }
\end{competences}

% %======================================================================
\sectionTitle{Work Experiences}{\faBriefcase}
% %======================================================================
\begin{experiences}
  \experience%
    [January 2023]%
    {Now}%
    {School of Computer Science and Engineering, Central South University}%
    [\begin{itemize}
      \item Lecturer
    \end{itemize}]
\end{experiences}

%======================================================================
\sectionTitle{Education}{\faGraduationCap}
%======================================================================
\begin{educations}
  \education%
    {September 2017}%
    [December 2022]%
    {Central South University}%
    {School of Computer Science and Engineering}%
    {Computer Science and Technology}%
    {PhD}

  \separator{0.5ex}
  \education%
    {November 2018}%
    [April 2019]%
    {Clemson University}%
    {School of Computing}%
    {Prof. Feng Luo's lab}%
    {Visiting scholar}

  \separator{0.5ex}
  \education%
    {September 2014}%
    [June 2017]%
    {Central South University}%
    {School of Information Science and Engineering}%
    {Computer Science and Technology}%
    {Master's Degree}

  \separator{0.5ex}
  \education%
    {September 2010}%
    [June 2014]%
    {Shandong University}%
    {School of Computer Science and Technology}%
    {Computer Science and Technology}%
    {Bachelor's Degree}
\end{educations}

%======================================================================
\sectionTitle{Research Achievements}{\faAtom}
%======================================================================
\begin{itemize}
    \item[>] Developed \link{https://github.com/bioinfomaticsCSU/deepsignal}{\it DeepSignal} and \link{https://github.com/PengNi/deepsignal-plant}{\it DeepSignal-plant} for 5mCpG detection in human and 5mC detection in plants from nanopore reads.
    \item[>] Developed \link{https://github.com/PengNi/ccsmeth}{\it ccsmeth} for 5mCpG detection in human, and a nextflow workflow \link{https://github.com/PengNi/ccsmethphase}{\it ccsmethphase} for methylation phasing from PacBio CCS reads.
    \item[>] Developed a Spark-based workflow \link{https://github.com/PengNi/basemods_spark}{\it basemods\_spark} for base modification detection using PacBio SMRT seqeuncing.
  \end{itemize}

%======================================================================
% Papers / Publications
\sectionTitle{Publications}{\faFileAlt}
%======================================================================
\begin{itemize}
  \item \textbf{First-Author}
  \begin{enumerate}
    %\small
    \item \textbf{Peng Ni}$^{\#}$, Fan Nie$^{\#}$, Zeyu Zhong, Jinrui Xu, Neng Huang, Jun Zhang, Haochen Zhao, You Zou, Yuanfeng Huang, Jinchen Li, Chuan-Le Xiao*, Feng Luo*, and Jianxin Wang*. \enquote{DNA 5-methylcytosine detection and methylation phasing using PacBio circular consensus sequencing.} {\it Nature Communications} 14 (2023): 4054. (co-first author; SCI; JCR Q1; IF=16.6)
    \item \textbf{Peng Ni}, Neng Huang, Fan Nie, Jun Zhang, Zhi Zhang, Bo Wu, Lu Bai, Wende Liu, Chuan-Le Xiao*, Feng Luo*, and Jianxin Wang*. \enquote{Genome-wide detection of cytosine methylations in plant from Nanopore data using deep learning.} {\it Nature Communications} 12 (2021): 5976. (SCI; JCR Q1; IF=16.6)
    % \bigskip
    % \medskip
    \item \textbf{Peng Ni}, Jianxin Wang*, Ping Zhong, Yaohang Li, Fang-Xiang Wu, and Yi Pan. \enquote{Constructing Disease Similarity Networks Based on Disease Module Theory.} {\it IEEE/ACM Transactions on Computational Biology and Bioinformatics} 17, no. 03 (2020): 906-915. (SCI; JCR Q1; IF=3.710)
    \item \textbf{Peng Ni}$^{\#}$, Neng Huang$^{\#}$, Zhi Zhang, De-Peng Wang, Fan Liang, Yu Miao, Chuan-Le Xiao*, Feng Luo*, and Jianxin Wang*. \enquote{DeepSignal: detecting DNA methylation state from Nanopore sequencing reads using deep-learning.} {\it Bioinformatics} 35, no. 22 (2019): 4586-4595. (co-first author; SCI; JCR Q1; IF=6.937)
  \end{enumerate}
  %\medskip
  \item \textbf{Other}
  \begin{enumerate}
    % \small
    \item Haochen Zhao, \textbf{Peng Ni}, Qichang Zhao, Xiao Liang, Di Ai, Shannon Erhardt, Jun Wang, Yaohang Li, and Jianxin Wang*. \enquote{Identifying the serious clinical outcomes of adverse reactions to drugs by a multi-task deep learning framework.} {\it Communications Biology} 6, no. 1 (2023): 870. (SCI; JCR Q1; IF=6.548)
    \item Neng Huang, Minghua Xu, Fan Nie, \textbf{Peng Ni}, Chuan-Le Xiao, Feng Luo, and Jianxin Wang*. \enquote{NanoSNP: a progressive and haplotype-aware SNP caller on low-coverage nanopore sequencing data.} {\it Bioinformatics} 39, no. 1 (2023): btac824. (SCI; JCR Q1; IF=6.937)
    \item Jun Zhang, Fan Nie, Neng Huang, \textbf{Peng Ni}, Feng Luo, and Jianxin Wang*. \enquote{Fec: a fast error correction method based on two-rounds overlapping and caching.} {\it Bioinformatics} 38, no. 19 (2022): 4629-4632. (SCI; JCR Q1; IF=6.937)
    \item Neng Huang, Fan Nie, \textbf{Peng Ni}, Xin Gao, Feng Luo*, and Jianxin Wang*. \enquote{BlockPolish: accurate polishing of long-read assembly via block divide-and-conquer.} {\it Briefings in bioinformatics} 23, no. 1 (2022): bbab405. (SCI; JCR Q1; IF=11.622)
    \item Neng Huang, Fan Nie, \textbf{Peng Ni}, Feng Luo, Xin Gao*, and Jianxin Wang*. \enquote{NeuralPolish: a novel Nanopore polishing method based on alignment matrix construction and orthogonal Bi-GRU Networks.} {\it Bioinformatics} 37, no. 19 (2021): 3120-3127. (SCI; JCR Q1; IF=6.937)
    \item Haochen Zhao, Guihua Duan, \textbf{Peng Ni}, Cheng Yan, Yaohang Li, and Jianxin Wang*. \enquote{RNPredATC: a deep residual learning-based model with applications to the prediction of drug-ATC code association.} {\it IEEE/ACM Transactions on Computational Biology and Bioinformatics} (2021). (SCI; JCR Q1; IF=3.710)
    \item Neng Huang, Fan Nie, \textbf{Peng Ni}, Feng Luo, and Jianxin Wang*. \enquote{Sacall: a neural network basecaller for oxford nanopore sequencing data based on self-attention mechanism.} {\it IEEE/ACM Transactions on Computational Biology and Bioinformatics} (2020). (SCI; JCR Q1; IF=3.710)
    \item Huimin Luo, Jianxin Wang*, Min Li, Junwei Luo, \textbf{Peng Ni}, Kaijie Zhao, Fang-Xiang Wu, and Yi Pan. \enquote{Computational Drug Repositioning with Random Walk on a Heterogeneous Network.} {\it IEEE/ACM Transactions on Computational Biology and Bioinformatics} 16, no. 06 (2019): 1890-1900. (SCI; JCR Q1; IF=3.710)
    \item Min Li*, \textbf{Peng Ni}, Xiaopei Chen, Jianxin Wang, Fang-Xiang Wu, and Yi Pan. \enquote{Construction of Refined Protein Interaction Network for Predicting Essential Proteins.} {\it IEEE/ACM Transactions on Computational Biology and Bioinformatics} 16, no. 04 (2019): 1386-1397. (SCI; JCR Q1; IF=3.710)
    \item Cheng Yan, Jianxin Wang*, \textbf{Peng Ni}, Wei Lan, Fang-Xiang Wu, and Yi Pan. \enquote{DNRLMF-MDA: Predicting microRNA-Disease Associations Based on Similarities of microRNAs and Diseases.} {\it IEEE/ACM Transactions on Computational Biology and Bioinformatics} 16, no. 01 (2019): 233-243. (SCI; JCR Q1; IF=3.710)
  \end{enumerate}
\end{itemize}

\end{document}
